build/linux_build.sh

```diff
@@ -59,32 +57,40 @@ cd "$RootPath"
 
 # C - Text Editor
 cp -f "include/webui.h" "examples/C/text-editor/webui.h"
-cp -f "build/Linux/GCC/webui-2-x64.so" "examples/C/text-editor/webui-2-x64.so"
+cp -f "$BuildPath/GCC/webui-2-x64.so" "examples/C/text-editor/webui-2-x64.so"
 
 echo "";
 if [ "$ARG1" = "" ]; then
 
-    echo "Copying WebUI libs to the release folder..."
+    echo "Copying WebUI libs to $DistPath..."
     echo "";
 
-    # Release Linux Include
-    cp -f "include/webui.h" "Release/Linux/include/webui.h"
-    cp -f "include/webui.hpp" "Release/Linux/include/webui.hpp"
+    # Remove Linux distributable files directory if it exits
+    [ -d "$DistPath" ] && rm -r "$DistPath"
+
+    # Create Linux output directories
+    mkdir -p "$DistPath/include"
+    mkdir "$DistPath/GCC"
+    mkdir "$DistPath/Clang"
```

Comment:
The release directories were a fixed part of the repository and their presence did not need to be checked. After having them removed: Here, we make sure the destinations the files are copied to exist and are clean.